src/ImageSharp/Formats/Png/PngEncoder.cs

```diff
@@ -18,7 +17,12 @@ public class PngEncoder : QuantizingImageEncoder
 
         // We set the quantizer to null here to allow the underlying encoder to create a
         // quantizer with options appropriate to the encoding bit depth.
-        this.Quantizer = null;
+        this.Quantizer = null!;
+
+    /// <summary>
+    /// Gets whether the file is a simple PNG.
+    /// </summary>
+    public bool? IsSimplePng { get; init; }
```

Comment:
No need for this. The encoder should automatically encode multiple frames as animated png.